static/docs/command-reference/import-url.md

```diff
@@ -79,8 +78,8 @@ DVC supports several types of (local or) remote locations (protocols):
 
 Another way to understand the `dvc import-url` command is as a short-cut for a
 more verbose `dvc run` command. This is discussed in the
-[External Dependencies](/doc/user-guide/external-dependencies) documentation,
```

Comment:
I think we should have external dependencies and external outputs sections still. They both should be introductory, provide some context about each.

Reply:
I don't understand this, what do you mean?